src/main/java/com/cairn/waypoint/dashboard/endpoints/HomeworkQuestionLinkedProtocolTemplate/AddHomeworkQuestionLinkedProtocolTemplateDto.java

```diff
@@ -0,0 +1,22 @@
+package com.cairn.waypoint.dashboard.endpoints.HomeworkQuestionLinkedProtocolTemplate;
+
+class AddHomeworkQuestionLinkedProtocolTemplateDto {
+    private Long protocolTemplateId;
+    private Long homeworkQuestionId;
+
+    public Long getProtocolTemplateId() {
```

Comment:
Let's remove these explicit getters and setters to utilize the paradigm I've setup in our other DTOs. Check them out and you will see a series of Lombok annotations that generate getters, setters, equals, hashCode, and other methods for our POJO classes